### ISSUE TITLE
Make grabbing objects connected with other objects not janky

### ISSUE THREAD

**Issue by bibbybabby** [studio-minus/ppg-bugs-and-requests#1509]
**Feature description**
Make it so when humans grab stuff the other objects connected by wire also move to their correct spots

**Feature purpose**
QOL because you need to move them manually in pause as of right now

**Additional context**

https://user-images.githubusercontent.com/104321411/174541280-7398091f-c571-4e6f-8886-37c4cdce9942.mp4

**Comment by DindinYT37** [comment]
I agree that this should be fixed but rather, it concerns me how you handle your humans.
